Runtime/NodeE2EServerApp/readme.md

```diff
@@ -1 +0,0 @@
-Moved to https://github.com/Azure/azure-mobile-apps-node/tree/master/test/e2e
```

Comment:
We want to keep these files, please make one in the old net test folder as well...